src/storage/exec/GetNeighborsNode.h

```diff
@@ -42,7 +42,11 @@ class GetNeighborsNode : public QueryNode<VertexID> {
     if (ret != nebula::cpp2::ErrorCode::SUCCEEDED) {
       return ret;
     }
-
+    if (context_->env()->metaClient_ &&
+        context_->env()->metaClient_->checkIsPlanKilled(context_->planContext_->sessionId_,
+                                                        context_->planContext_->planId_)) {
```

Comment:
Why not provide a function to check whether current plan is killed in BaseNode? such as:

```cpp
bool isPlanKilled() {
  auto client = context_->env()->metaClient_ ;
  auto ctx = context_->planContext_;
  return client && client->checkIsPlanKilled(ctx->sessionId_, ctx->planId_)
}
```